### PR TITLE
Update DeprecatedWorkbookRetrievalTool.md with raw links

### LINKED ISSUE #1562: Cannot load Deprecated Workbook Retrieval Tool correctly
Hi!
I am trying to run the Deprecated Workbook Retrieval Tool as described here - https://github.com/microsoft/Application-Insights-Workbooks/blob/master/Documentation/LegacyAI/DeprecatedWorkbookRetrievalTool.md. 

When I try to load it in a new Workbook, I get an error. After further review, there are unnecessary double quotes in some locations. After I removed the double quotes (Using Visual Studio Code) it loads. After selecting a Subscription and Resource Group though, another section has an error where it says an API parameter must be added and the Subscription path was not passed. I added the subscription path manually and a API, but nothing else loads.

Can you take a look please?

Thank you,
Jeff Kozloff

### PR DESCRIPTION
**Pull request by pizzaz93**
Fixes #1562

### CODE REVIEW

**Comment by pizzaz93** [comment]
#assign-reviewer: @gardnerjr

**Review by gardnerjr — APPROVED** [review]

**Comment by pizzaz93** [comment]
@gardnerjr I think it is waiting for a workflow approval.

**Comment by pizzaz93** [comment]
Closing due to lack of activity.

**Pull request closed without merging**
